### PR TITLE
BAH-2811 | registration card print feature for clinics

### PR DESCRIPTION
**Pull request by kavitha-sundararajan**

### CODE REVIEW

File: openmrs/i18n/registration/locale_pt_BR.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 {		
 	"REGISTRATION_SEARCH_CONFIG_KEY": "P<u>e</u>squisar",	
-	"REGISTRATION_CREATE_NEW_CONFIG_KEY":"Criar <u>N</u>ovo"
+	"REGISTRATION_CREATE_NEW_CONFIG_KEY": "Criar <u>N</u>ovo",
+	"REGISTRATION_PRINT_REG_CARD_KEY": "<u>I</u>mprimir Cart�o Registro",	
+	"REGISTRATION_BRING_CARD_TEXT_KEY": "� muito importante trazer o cart�o para o hospital.",
```
Comment (gsluthra) [inline openmrs/i18n/registration/locale_pt_BR.json]: Some characters seem messed up to me on github. Does it look fine to you on your browser?

File: openmrs/i18n/registration/locale_en.json [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
-  "REGISTRATION_CREATE_NEW_CONFIG_KEY":"Create <u>N</u>ew",
-  "OBS_ABNORMAL_TRANSLATE_KEY":"Abnormal",
+  "REGISTRATION_CREATE_NEW_CONFIG_KEY": "Create <u>N</u>ew",
+  "OBS_ABNORMAL_TRANSLATE_KEY": "Abnormal",
   "NUTRITIONAL_VALUES_LOCALE_KEY": "Nutritional Values",
   "FEE_INFORMATION_LOCALE_KEY": "Fee Information",
-  "REGISTRATION_MESSAGE": "Thank you for registering at #clinicName. Please note your Patient Registration Details: #patientId, #name, #gender, #age years. For any queries call us on #helpDeskNumber. Clinic Timings: All weekdays, 10 am - 7 pm. Weekends closed."
+  "REGISTRATION_MESSAGE": "Thank you for registering at #clinicName. Please note your Patient Registration Details: #patientId, #name, #gender, #age years. For any queries call us on #helpDeskNumber. Clinic Timings: All weekdays, 10 am - 7 pm. Weekends closed.",
+  "REGISTRATION_PRINT_REG_CARD_KEY": "Print Registration Card",
+  "REGISTRATION_BRING_CARD_TEXT_KEY": "It is quite important to bring the card to the hospital.",
```
Comment (gsluthra) [inline openmrs/i18n/registration/locale_en.json]: Instead put: "Please bring this card with you every time you visit for a faster service experience."  (remove word hospital/clinic from this message).

File: openmrs/i18n/registration/locale_en.json [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
+  "REGISTRATION_TAKE_MEDICINES_TEXT_KEY": "Take medicines according to instructions of the doctor.",
+  "REGISTRATION_TIMINGS_TEXT_KEY": "Registration Timings are from 8:30 am to 12:00 pm.",
+  "REGISTRATION_OPD_DAYS_TEXT_KEY": "OPD days are on Monday, Wednesday, Friday.",
+  "REGISTRATION_SUNDAYS_HOLIDAY_TEXT_KEY": "Sundays are holidays.",
+  "REGISTRATION_SMOKING_INJURIOUS_TEXT_KEY": "Smoking, Tobacco chewing can cause cancer, heart diseases and ASTHMA.",
```
Comment (gsluthra) [inline openmrs/i18n/registration/locale_en.json]: `Smoking & tobacco chewing can cause Cancer, Heart disease and Asthma.` (use correct case)

File: openmrs/i18n/registration/locale_pt_BR.json (hunk 1 above)
Comment (kavitha-sundararajan) [reply openmrs/i18n/registration/locale_pt_BR.json]: Sorry, it is my editor issue. Refactored now

File: openmrs/i18n/registration/locale_en.json (hunk 2 above)
Comment (kavitha-sundararajan) [reply openmrs/i18n/registration/locale_en.json]: Updated to the given text

File: openmrs/i18n/registration/locale_en.json (hunk 3 above)
Comment (kavitha-sundararajan) [reply openmrs/i18n/registration/locale_en.json]: Updated the given text

**Review by gsluthra — APPROVED** [review]

**Pull request merged**